activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -148,21 +152,21 @@ public void init() {
                                                      deploymentConverter,
                                                      configuration,
                                                      eventPublisher,
-                                                     processVariablesValidator));
+                                                     processVariablesValidator,
+                                                     securityManager));
 
         processAdminRuntimeMock = spy(new ProcessAdminRuntimeImpl(repositoryService,
                                                      processDefinitionConverter,
                                                      runtimeService,
                                                      processInstanceConverter,
+                                                     variableInstanceConverter,
                                                      eventPublisher,
                                                      processVariablesValidator));
 
         //Reset test variables
         RuntimeTestConfiguration.processImageConnectorExecuted = false;
         RuntimeTestConfiguration.tagImageConnectorExecuted = false;
         RuntimeTestConfiguration.discardImageConnectorExecuted = false;
-
-        securityUtil.logInAs("user");
```

Comment:
don't we need to change/add any test?

Reply:
I removed because its a duplicate of line 134 https://github.com/Activiti/Activiti/blob/develop/activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java#L139

Reply:
I see from the conflict page it was not clear. I really hope that it was not added to cover some magic :p